src/lib/containers/HasAccessV2.tsx

```diff
@@ -111,11 +111,11 @@ export function useGetFileHandleDownloadType(
   const { accessToken } = useSynapseContext()
   const { data: entityBundle, error: entityFetchError } = useGetEntityBundle(
     entityId,
+    parseInt(entityVersionNumber ?? ''),
```

Comment:
Is NaN (parseInt('') handled properly by useGetEntityBundle()?

Reply:
Not sure, I'll change it to make it more clear/safe